std_msgs/Header header
uint8 max_battery
---
std_msgs/Header header
bool goal_reached
---
std_msgs/Header header
uint8 current_battery
